lib/v2/cankaoxiaoxi/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'cankaoxiaoxi.com': {
+        _name: '参考消息',
+        '.': [
+            {
+                title: '分类',
```

Comment:
```suggestion
                title: '新闻分类',
```
This should match H3 heading of the docs